src/integration-test/java/io/aiven/kafka/connect/opensearch/AbstractKafkaConnectIT.java

```diff
@@ -111,7 +111,7 @@ Map<String, String> connectorProperties() {
         return props;
     }
 
-    void writeRecords(final int numRecords) {
+    void writeRecords(final int numRecords, String topicName) {
```

Comment:
We have conflict with `topicName` field here, making method static would eliminate the confusion:

```suggestion
    static void writeRecords(final int numRecords, String topicName) {
```

Reply:
Updated the param name, instead of changing the method to static, as changing to static needs to change 'connect' to static as well